target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,366 @@
+// SPDX-License-Identifier: MIT, GPL-2.0 or later
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+#include "ipq6018.dtsi"
```

Comment:
`ipq6018-512m.dtsi` is enough as it will include `ipq6018.dtsi`

Reply:
> `ipq6018-512m.dtsi` is enough as it will include `ipq6018.dtsi`

removed